py/src/test_call.py

```diff
@@ -774,19 +970,17 @@ def test_failing_mainnet_tx2():
 
     print(_timings)
 
-    # this is correct in 0.9.1
+    # this is correct in 0.10, not in 0.9.1
     assert output == {
-        "gas_consumed": 10083,
+        "gas_consumed": 10102,
         "gas_price": 21367239423,
-        "overall_fee": 215446943464081,
+        "overall_fee": 215851852651146,
     }
 
-    assert output["overall_fee"] == 0xA9B3FBAC7457
-
```

Comment:
This was a leftover 0.9.0 value, forgotten, because the tests are ignored by default.